README.md

```diff
@@ -46,19 +47,47 @@ const checkoutInstance = startCheckout(manifest, [options])
 checkoutInstance.unmount()
 ```
 
-### On Completion
+### On Success
+
+```javascript
+function mySuccessHandler(checkoutPaymentInfo) {
+  checkoutInstance.unmount()
+  if (checkoutPaymentInfo.paid) {
+    document.write('Your payment was received. Thank you.')
```

Comment:
Instead of show `document.write`, just set a comment like this `/** Payment received */` to indicate the user that is where he should manage this behaviour, and on else `/** Order received. Awaiting payment */`.